### DISCUSSION TITLE
Latex doesn't appear correctly in the app

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18128] [Mobile] [answered]
but it appears perfectly in the browser.
example:
$$ \LaTeX $$
(I'm using the ios client)
will latex support be added in the future?

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback @3m4r5 — I've opened an issue for our team to track progress on this, plus a handful of other content types that don't quite render correctly on mobile.

**Reply by 3m4r5** [reply]
thank you, I hope they get added.

**Reply by foxyseta** [reply]
Same here:
![Screenshot_20220728-212425.png](https://user-images.githubusercontent.com/12249077/181621152-1c7e47f7-e619-49b7-8215-c29c5ba70ea3.png)
![Screenshot_20220728-212328.png](https://user-images.githubusercontent.com/12249077/181621246-f53ae3f4-adde-4026-9c4c-efb6c210f5da.png)
